app/vaccine_warrior.py

```diff
@@ -26,6 +28,18 @@
 openfile = open(abs_file_path)
 facility_list = json.load(openfile)
 
+
```

Comment:
Can we please remove this block of commented-out lines of facility_list to avoid any confusion?  These records showcase the bug of availability=FIRST DOSE.. which we already fixed.

Reply:
Sure, these two lines were used to import vaccine stops locally from facility_data.json. But now the information is imported from website_scraper, we don't need to import locally anymore.